media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -142,21 +138,21 @@ <h1>Ochrana osobných údajov</h1>
 Súhlas so zverejnením osobných údajov je dotknutá osoba oprávnená kedykoľvek odvolať.
 </p>
 
-<p>
+<h3>
 <span>8.</span>
-TRETIE KRAJINY
-</p>
+Tretie krajiny
+</h3>
 
 <p>
 <span>8.1.</span>
 K prenosu osobných údajov dotknutej osoby do tretích krajín, t.j. do krajín, ktoré nie sú členskými
 štátmi Európskej Únie alebo zmluvnou stranou Dohody o Európskom hospodárskom priestore nedochádza.
 </p>
 
-<p>
+<h3>
 <span>9.</span>
-ĎALŠIE PRÁVA DOTKNUTEJ OSOBY
-</p>
+Ďalšie práva dotknutej osoby
+</h3>
```

Comment:
Podkapitola 9.2.2. chyba text: "... priameho marketingu v poštovom styku, alebo"

Reply:
@viliambalaz Nerozimiem, kde chyba text? Ak mas na mysli za "alebo", tak nie tam text nechyba. V pravnych textoch sa takto pisu odrazky. Ze text za "alebo" pokracuje v dalsom odstavci.

Takze taketo 4 odseky:

```
9.2. Dotknutá osoba má právo (...) namietať voči:

9.2.1. spracúvaniu jej osobných údajov (...);

9.2.2. využívaniu osobných údajov (...), alebo

9.2.3. poskytovaniu osobných údajov (...).
```

si predstav ako jednu vetu: "Dotknutá osoba má právo ... namietať voči: spracúvaniu jej osobných údajov (...); využívaniu osobných údajov (...), alebo poskytovaniu osobných údajov (...)." Akurat je to rozpisane do odstavcov.